docs/curators/risk-reports/2681185522664180117929158590481443496806090795357786716961716864181408932939/dog.mdx

```diff
@@ -0,0 +1,165 @@
+
+The pool enables various Starknet assets to be borrowed and used as collateral.
+
+It uses Vesu's official factory extension with Pragma price feeds.
+
+The market's overall risk rating is 🟩 Low.
+
+*Datapoints*
+
+- [CoinGecko](https://www.coingecko.com/en/coins/starknet)
+- [Voyager](https://voyager.online/contract/0x40e81cfeb176bfdbc5047bbc55eb471cfab20a6b221f38d8fda134e1bfffca4)
+
+
+## Pool Risk
+
+The *CarmineDAO Runes* pool is curated by CarmineDAO, a DAO focused on providing tools to monitor and manage risks in DeFi. DAO whose members work and worked on projects like DeRisk.
+
+The pool uses Vesu's audited factory extension which leverages a robust Pragma feed and Vesu's public liquidation model.
+
+CarmineDAO has the permission to pause the pool in an emergency situation and update various parameters including debt caps and liquidation loan-to-value ratios. 
+
+Given CarmineDAO's and its members expertise in assessing, monitoring and managing DeFi risks including managing and monitoring liquidity in the market and a multisig owning the pool permissions, the pool risk is deemed low.
+
+*Datapoints*
+
+- [CarmineDAO's discord](https://discord.com/invite/uRs7j8w3bX)
+- [DeRisk](https://www.carmine.finance/derisk)
+- [Extension Docs](https://docs.vesu.xyz/)
+- [Audit Reports](https://github.com/vesuxyz/security)
+- [Source Code](https://github.com/vesuxyz/vesu-v1/blob/main/src/extension/default_extension_po.cairo)
+
+**Rating**
+
+The proposed rating is:
+
+- 🟩 Low
+
+
+## Asset Risk
+
+:::warning
+This report does not assess the risks associated with bridging assets to Starknet using StarkGate.
+Make sure you understand the risks involved with bridging assets to Starknet before continuing.
+:::
+
+StarkGate DOG reflects Bitcoin's DOG that is bridged to Starknet using the StarkGate bridge. 
+StarkGate is Starknet's canonical bridge and is developed and maintained by StarkWare, the creator of Starknet.
+The bridge is operational since early 2022 and has facilitated over [1.9M bridge transactions](https://dune.com/sapiens/starknet-insighttokenomics-and-bridge-stats) or over USD 1.7bn in value bridged.
+
+**Smart contract risk**
+
+StarkGate DOG uses OpenZeppelin's [standard ERC20](https://github.com/OpenZeppelin/cairo-contracts) token implementation.
+It has been audited by Nethermind and both [source code](https://github.com/starknet-io/starkgate-contracts) and [audit report](https://github.com/NethermindEth/PublicAuditReports/blob/main/NM0064-DRAFT_STARKGATE.pdf) are publicly available.
+
+:::warning
+The StarkGate DOG contract is not verified on block explorers.
+:::
+
+**Counterparty risk**
+
+N/A
+
+**Depeg risk**
+
+N/A
+
+**Datapoints**
+
+- [Token Contract](https://voyager.online/contract/0x40e81cfeb176bfdbc5047bbc55eb471cfab20a6b221f38d8fda134e1bfffca4)
+- [Mainnet Escrow Contract](https://etherscan.io/address/0xae0Ee0A63A2cE6BaeEFFE56e7714FB4EFE48D419)
```

Comment:
this Ethereum escrow contract is not linked to the DOG bridge, can you link to the bitcoin escrow?